reference/features/distinct.md

```diff
@@ -38,13 +38,15 @@ As shown below, you have 3 documents that contain information about the same jac
 ]
 ```
 
-You may want to ignore the different colors of an item. To do so, you can set `product_id` as a `distinctAttribute`.
+If a user searched for a `lee leather jacket`, MeiliSearch would return all three instances by default. This could cause the results to be cluttered with almost identical variations of the same item.
```

Comment:
```suggestion
By default, a search for `lee leather jacket` would return all three documents. This might not be desired, since displaying nearly identical variations of the same item can make results appear cluttered.
```